example/mnist-embeddings.py

```diff
@@ -14,9 +14,15 @@
 from PIL import Image  # type: ignore
 import base64 as b64  # type: ignore
 from six import BytesIO
+import sys
+
+try:
+    features = np.loadtxt("example/data/mnist2500_X.txt")
+    labels = np.loadtxt("example/data/mnist2500_labels.txt")
+except:
+    print("Unable to find files mmist2500_X.txt and mnist2500_labels.txt in the example/data/ directory. Please download from https://github.com/lvdmaaten/lvdmaaten.github.io/blob/master/tsne/code/tsne_python.zip")
```

Comment:
I feel like we should conditionally ask the user if they want to download and then do it ourselves, but this works fine for now

Reply:
+1, I'll slot that as an enhancement after this lands.